modules/machine-edge-pool-review-nodes.adoc

```diff
@@ -0,0 +1,56 @@
+
+:_content-type: PROCEDURE
+[id="machine-edge-pool-review-nodes_{context}"]
+= Verifying nodes that were created with edge compute pool
+
+After your install a cluster that uses AWS Local Zones, check the status of the machine that was created by the machine set manifests created at install time.
+
+. To check the machine sets created from the subnet you added to the `install-config.yaml` file, enter the following command:
```

Comment:
Change to: "..., run the following command:"

Reply:
```suggestion
. To check the machine sets created from the subnet you added to the `install-config.yaml` file, run the following command:
```